geometry_msgs/Point offset
string frame_id
float32 min_range
float32 max_range
float32 min_angle
float32 max_angle
float32 resolution